src/seedu/addressbook/Main.java

```diff
@@ -1,8 +1,9 @@
 package seedu.addressbook;
 
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Hi, this line is a coding violation. Packages you use should be stated explicitly.